src/wallet/adapters/ConnectAdaptor.ts

```diff
@@ -275,13 +271,16 @@ export class ConnectAdaptor implements AUTHAdapter {
   }
 
   async waitWebAuthnSignerDeployment(
+    publicKey_Id: string,
     publicKey_X: string,
     publicKey_Y: string
   ): Promise<void> {
-    if (!this.projectParams) throw new Error('No project Params found')
+    const webAuthnSigner = await this.API.getWebAuthnSignerByPublicKeyId(
```

Comment:
Need a call ? Cannot be passed by the called ? All infos has not already returned been to the front ?